src/components/projects/Intro.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+export default function Intro() {
+  return (
+    <div className={`flex flex-col text-center font-monda w-[56%]`}>
```

Comment:
do not do a fixed width like that. Use w-7/12 or any value out of 12 instead. Other than that good to go!

Reply:
I had it at w-7/12, but it wasn't at the exact measurements of the given photo. It isn't a drastic change to the text other than placement of words, would you still like it?

Reply:
yess that is okay. Could you add text-lg for the font size, leading-8 for the line height and w-8/12 for the width. And can you import your component in the app/projects/page.js file